doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -68,10 +68,51 @@ Note that each requirement may have one or more test cases associated with it.
 
 <a name="4.3.4"></a>
 ### 4.3.4 Interfaces & APIs
-This defines the test cases around the functionality that are exposed by OpenStack APIs. All the defined OpenStack
-service APIs in RA-1's [chapter 05](../../../ref_arch/openstack/chapters/chapter05.md) will be the scope here.
 
-Note: It will only target the functionality that are exposed by standard OpenStack APIs.  
+The [OpenStack Gates](https://opendev.org/openstack/devstack-gate) verify all
+changes proposed mostly by running thousands of Tempest tests completed by
+Rally scenarios in a few cases. Skipping tests is allowed in all OpenStack
+Gates and only failures rate the review -1 because of the multiple capabilities
+and backends selected in the different Gate jobs. The classical
+[Functest containers](https://wiki.opnfv.org/pages/viewpage.action?pageId=29098314)
+conform to this model which also fits the heterogeneous user deployments.
+
+From a CNTT Compliance state point, the capabilities are well described in
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
```

Comment:
Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`

Reply:
unclear and requires details (TSC?). Most of links in doc are absolute and it seems worse to start with ../../../ as the few relative dirs in tree.

Reply:
I strongly which shouldn't block the chain here for that.
(All the links would be changed in one commit if the rule is agreed).

Reply:
@collivier Relative works on both github md and compiled html. External links has to be absolute. Intent is that if a person is viewing it on github, should go to md files on click, and person viewing it on github.io should go to html.
Can raise it to TSC and add.

@rabi-abdel @rabiabdel Comments?

Reply:
@deebhatia  Would you mind merging all patches which conform with Jekyll?
We do modify all links before the release whatever the TSC decision.
Relative links can be more quickly broken and mislead easily.

Reply:
I totally agree. Resolved.